apps/fulfillment-e2e/cypress.config.js

```diff
@@ -23,5 +23,7 @@ module.exports = defineConfig({
   },
   env: {
     ORYX_FULFILLMENT_BACKEND_URL: process.env.ORYX_FULFILLMENT_BACKEND_URL,
+    ORYX_FULFILLMENT_PUBLIC_VAPID_KEY:
```

Comment:
Why do we need this variable here? (We have discussed it with @tarashaliuk and it should have been removed)

Reply:
Probably some issues after merging to the dev Taras's branch. This branch one was created based on his one. Dropped